src/hooks/cakeVault/useGetVaultBountyInfo.ts

```diff
@@ -2,44 +2,36 @@ import { useState, useEffect } from 'react'
 import BigNumber from 'bignumber.js'
 import { useGetApiPrice } from 'state/hooks'
 import { useCakeVaultContract } from 'hooks/useContract'
-import { getFullDisplayBalance } from 'utils/formatBalance'
 import makeBatchRequest from 'utils/makeBatchRequest'
 import { getCakeAddress } from 'utils/addressHelpers'
 
 const useGetVaultBountyInfo = (refresh?: number) => {
   const cakeVaultContract = useCakeVaultContract()
-  const [estimatedCallBountyReward, setEstimatedCallBountyReward] = useState(null)
-  const [totalPendingCakeRewards, setTotalPendingCakeRewards] = useState(null)
-  const [dollarCallBountyToDisplay, setDollarBountyToDisplay] = useState(null)
-  const [cakeCallBountyToDisplay, setCakeBountyToDisplay] = useState(null)
+  const [estimatedDollarBountyReward, setEstimatedDollarBountyReward] = useState(null)
+  const [estimatedCakeBountyReward, setEstimatedCakeBountyReward] = useState(null)
+  const [totalPendingCakeHarvest, setTotalPendingCakeHarvest] = useState(null)
 
   const cakePrice = useGetApiPrice(getCakeAddress())
 
   useEffect(() => {
-    // Call contract to get estimated rewards
     const fetchRewards = async () => {
-      const [estimatedRewards, pendingCakeRewards] = await makeBatchRequest([
+      const [estimatedClaimableCakeReward, pendingTotalCakeHarvest] = await makeBatchRequest([
         cakeVaultContract.methods.calculateHarvestCakeRewards().call,
         cakeVaultContract.methods.calculateTotalPendingCakeRewards().call,
       ])
-      setEstimatedCallBountyReward(new BigNumber(estimatedRewards as string))
-      setTotalPendingCakeRewards(new BigNumber(pendingCakeRewards as string))
+      if (cakePrice) {
+        const dollarValueOfClaimableReward = new BigNumber(estimatedClaimableCakeReward as string).multipliedBy(
+          cakePrice,
+        )
+        setEstimatedDollarBountyReward(dollarValueOfClaimableReward)
+      }
+      setEstimatedCakeBountyReward(new BigNumber(estimatedClaimableCakeReward as string))
+      setTotalPendingCakeHarvest(new BigNumber(pendingTotalCakeHarvest as string))
     }
     fetchRewards()
-  }, [cakeVaultContract, refresh])
+  }, [cakeVaultContract, cakePrice, refresh])
```

Comment:
FYI, cakePrice already refresh according to the slowRefresh rule

Reply:
Ah sure, cheers Doge. This is the component - we need to fetch the harvestable cake rewards on a fast refresh cycle, as they often change block-to-block. 

<img width="272" alt="Screenshot 2021-05-05 at 08 53 40" src="https://user-images.githubusercontent.com/79279477/117111381-69a7f980-ad7f-11eb-82a8-44c2b50631d2.png">

I'm not 100% why I was passing the refresh into the hook, rather than declaring it within the hook though. I've amended that